### ISSUE TITLE
juniper.device.jsnapy -  save the name of the failed tests to a dest_dir path .

### ISSUE THREAD

**Issue by chidanandpujar** [Juniper/ansible-junos-stdlib#676]
Issue Type
------

 - Feature Idea

Module Name
------
juniper.device.jsnapy

juniper.device collection and Python libraries version

```
Collection                               Version
---------------------------------------- -------
juniper.device                           1.0.5  

jsnapy                1.3.8.dev0

```

OS / Environment
------
JUNOS Evolved

Summary
------
```
 list the name of the tests that failed when jsnapy has multiple tests in addition to the result already collected as below:
"total_failed": 3, >>>> save a list of the name of the failed tests to a dest_dir path .
"total_passed": 37,
"total_tests": 40
}
}
```

**Comment by chidanandpujar** [comment]
Fixed via #678

**Comment by chidanandpujar** [comment]
#678  Fix is under review

**Comment by chidanandpujar** [comment]
Fixed via #678
